pkg/kn/commands/service/delete.go

```diff
@@ -66,7 +89,21 @@ func NewServiceDeleteCommand(p *commands.KnParams) *cobra.Command {
 			return nil
 		},
 	}
+	flags := serviceDeleteCommand.Flags()
+	flags.BoolP("all", "", false, "Delete all Services in a namespace.")
```

Comment:
```suggestion
	flags.Bool("all", false, "Delete all services in a namespace.")
```

Reply:
Thanks! Addressed this and added e2e test.